scripts/install_vulkan_headers.sh

```diff
@@ -1,8 +1,54 @@
 #!/bin/bash
 
-git clone https://github.com/KhronosGroup/Vulkan-Headers.git "${temp_dir}/Vulkan-Headers"
+#--- Script for installing Vulkan Headers ---#
+
+
+### Setup ###
+
+## Source the default values for imported variables
+. scripts/default_values.sh
+
+## Source the global helper functions
+. scripts/system_utils.sh
+
+## Source the configurations for our dependencies
+. deps.sh
+
+
+### Package metadata setup ###
+
+if [ -z "${dep_name}" ]; then
+    dep_name="${dep_name_vulkan}"
+fi
+
+if [ -z "${src_dir}" ]; then
+    src_dir="${parent_dir_vulkan}/${dep_name_vulkan}"
+fi
+
+if [ -z "${dep_ver}" ]; then
+    dep_ver="${dep_ver_vulkan}"
+fi
+
+repo_url="https://github.com/KhronosGroup/Vulkan-Headers.git"
+build_dir="${src_dir}/build"
+#prefix_dir="install" # Override default
```

Comment:
Required?

Reply:
Nope, but left it in because of the uncertainty about comment above:
> Wait. Where was this being installed previously?